buildSrc/src/main/groovy/GradlePlugins.groovy

```diff
@@ -0,0 +1,5 @@
+class GradlePlugins {
```

Comment:
Exactly @devisnik. This is just a cosmetic addition to have `gradlePlugins` vs `GradlePlugins` in our buildscripts.